index.md

```diff
@@ -194,11 +194,13 @@ and our administrator may contact you if we need any extra information.</h4>
 <p><a href="{{ site.swc_pre_survey }}{{ site.github.project_title }}">Pre-workshop Survey</a></p>
 <p><a href="{{ site.swc_post_survey }}{{ site.github.project_title }}">Post-workshop Survey</a></p>
 {% elsif site.carpentry == "dc" %}
-  <p>Please be sure to complete these surveys before and after the workshop.</p>
+<p>Please be sure to complete these surveys before and after the workshop.</p>
 <p><a href="{{ site.dc_pre_survey }}{{ site.github.project_title }}">Pre-workshop Survey</a></p>
 <p><a href="{{ site.dc_post_survey }}{{ site.github.project_title }}">Post-workshop Survey</a></p>
-{% elsif site.carpentry == "lc" %}
-<p>Ask your instructor about pre- and post-workshop Survey details.</p>
+{% elsif page.carpentry == "lc" %}
+<p>Please be sure to complete these surveys before and after the workshop.</p>
```

Comment:
Please refactor the if-else clause to avoid duplication of the HTML. the if-else should be only for the `href`.